docs/guides/code_examples/open_telemetry/instrument_crawler.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from opentelemetry.trace import set_tracer_provider
+
+from crawlee._types import BasicCrawlingContext
+from crawlee.crawlers import ParselCrawler, ParselCrawlingContext
+from crawlee.otel.crawler_instrumentor import CrawlerInstrumentor
```

Comment:
when there is an export in the init py

```suggestion
from crawlee.otel import CrawlerInstrumentor
```